core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -19,13 +19,28 @@
 oppia.controller('StateEditor', [
```

Comment:
Yeah, it could be renamed, but there is no StateEditor directive as such, right?
The new directive would be question editor directive (which is really just a single state).

Reply:
Well, that's exactly it: you have ExplorationEditorTab and QuestionEditor controllers, but they both contain the reusable StateEditor directive. Right?

Reply:
Oh yes, definitely! That's the point of it being reusable.

Reply:
Done, renamed to ExplorationEditorTab.